fpm/build-deb

```diff
@@ -56,4 +54,7 @@ fpm -t deb \
     --description "Soundnode App is an Open-Source project to support Soundcloud for desktop Mac, Windows, and Linux." \
     --after-install "fpm/create-symlink" \
     --after-remove "fpm/remove-symlink" \
-    --deb-no-default-config-files
+    --deb-no-default-config-files \
+    "$DIST_DIR/.=/usr/share/soundnode/" \
+    "app/soundnode.png=/usr/share/pixmaps/" \
```

Comment:
@JonasGroeger should this be /usr/share/icons, or is this a more distro neutral approach?